bats/fb-test-katello.bats

```diff
@@ -4,6 +4,7 @@
 set -o pipefail
 
 @test "check hammer ping" {
+  sleep 60
```

Comment:
A slightly more advanced version (based on https://stackoverflow.com/questions/12321469/retry-a-bash-command-with-timeout#12321815) could be:

```bash
@test "check hammer ping" {
  local next_wait_time=0
  until hammer ping || [ $next_wait_time -eq 12 ]; do
     sleep $(( next_wait_time++ ))
  done
}
```